.travis.yml

```diff
@@ -145,7 +145,7 @@ matrix:
         # unstable, we combine them into a single unstable build that we can
         # mark as an allowed failure below.
         - os: linux
-          env: NUMPY_VERSION=dev SETUP_CMD='test --remote-data'
+          env: PYTHON_VERSION=3.7 NUMPY_VERSION=dev SETUP_CMD='test --remote-data'
```

Comment:
Travis syntax limitation is that the allowed to fail jobs has to be repeated both here and down at the `allow_failures` part, too. please add the modification there, too.

Reply:
I thought I did but obviously didn't. Either I overlooked or something weird happened during rebase. Anyway, hopefully it is fixed now. Good catch!